### DISCUSSION TITLE
Post Test 2

### DISCUSSION THREAD

**Discussion by giscus** [parker1609/parker1609.github.io#1] [Announcements]
# Post Test 2

테스트 코드 관련 질문 목차 Q. 테스트 코드의 장단점 Q. 좋은 테스트 코드 Q. TDD(Test-Driven Development) Q. Spring Boot를 활용한 API 서버 테스트 코드 Q. BDD Q. 테스트 코드의 장단점 제가 경험했던 테스트 코드의 장점은 세 가지입니다.
기능에 대한 확신을 할 수 있어 불안감을 해소해줍니다. 리팩토링한 후 바로 테스트 코드로 확인할 수 있습니다. 테스트 코드를 통해 프로덕션 코드를 이해할 수 있습니다. 테스트 코드의 단점은 비용입니다. 테스트 코드에 익숙하지 않을수록 테스트 코드에 대한 비용은 커지고 부담스러워집니다.

https://parker1609.github.io/post/post-test-2/

**Comment by parker1609** [comment]
댓글 테스트!!

**Comment by parker1609** [comment]
- 엄격한 제목 설정 후 테스트~
